apps/dashboard/src/contract-ui/tabs/claim-conditions/components/snapshot-upload.tsx

```diff
@@ -266,8 +266,11 @@ export const SnapshotUpload: React.FC<SnapshotUploadProps> = ({
                       <>
                         <Text as={ListItem}>
                           Files <em>must</em> contain one .csv file with a list
-                          of addresses and their <Code>maxClaimable</Code>.
-                          (amount each wallet is allowed to claim)
+                          of addresses and their{" "}
+                          <code className="rounded bg-gray-200/20 p-1 text-black dark:text-white">
```

Comment:
Also use the InlineCode component here

Reply:
urgg nice catch ser. it was my demo code